tests/test_flows_seed.py

```diff
@@ -89,42 +91,181 @@ def test_invalid_mnemonic(self):
         self.run_sequence(sequence)
 
 
-    def test_export_xpub_flow(self):
+    def test_export_xpub_standard_flow(self):
         """
-            Selecting "Export XPUB" from the SeedOptionsView should enter the Export XPUB flow and end at the SeedOptionsView.
+            Selecting "Export XPUB" from the SeedOptionsView should enter the Export XPUB flow and end at the MainMenuView
         """
+
+        def test_standard_xpubs(sig_tuple, script_tuple, coord_tuple):
```

Comment:
Minor note: since the `test` prefix is a keyword for `pytest` to pickup tests, it's a tiny bit confusing/misleading to use the same naming convention in an internal function that `pytest` obviously will not try to run as an independent test.

In some cases I've used a leading underscore to specially identify internal functions, which is slightly unnecessary (no point in communicating "HANDS OFF!" if it's an internal function) but I also kind of like because my brain naturally just says, "Oh, this is an internal thing" when I see it invoked).

tldr: Just don't start the internal function name with "test" and we've settled at least 95% of the issue.

Reply:
Ty, yes, I see it now!